pkg/service/did/ion.go

```diff
@@ -120,6 +117,7 @@ func (h *ionHandler) CreateDID(ctx context.Context, request CreateDIDRequest) (*
 			Purposes: []ion.PublicKeyPurpose{ion.Authentication, ion.AssertionMethod},
 		},
 	}
+	pubKeys = append(pubKeys, opts.PublicKeys...)
```

Comment:
I'm not sure we should allow adding any public keys without a signature. I'd recommend changing it to a `SignedPublicKeyRequest` of some sort.

Reply:
> I'm not sure we should allow adding any public keys without a signature.

Would you mind explaining the rationale? Is the intention here to ensure that the requester has control of the private key associated with the public key that's being added?

If so, is it ok to add this as a separate feature? There are details I think would need to be ironed out before doing so including: should the signature be of a hardcoded value or of a nonce? If the latter, how will the nonce be generated and communicated?

Reply:
> Would you mind explaining the rationale? Is the intention here to ensure that the requester has control of the private key associated with the public key that's being added?

Yes exactly.

> If so, is it ok to add this as a separate feature? 

Sure, let's open an issue.